app/assets/stylesheets/layouts/starry.scss

```diff
@@ -1,85 +1,217 @@
-a:visited { color: #484357; }
-
-.flash.success, .flash.error, .flash.inbox { background-color: #9692A8; }
-.pagination .current { background-color: #484357; }
-.pagination a { color: #F3F3F3; }
-.pagination a:visited { color: #9C9AA4; }
-.pagination .disabled { color: #58546B; }
+
+$bg_color_selectionpopup: #484357; // selection popups like character/alias on posts#editor
+$font_color_selectionpopup: #DADADA;
+$bg_color_expander: #9692A8; // expander on posts
+
+.selector-popup { background-color: $bg_color_selectionpopup; color: $font_color_selectionpopup; }
+.post-expander { background-color: $bg_color_expander; }
+
+$bg_color_post_icon: #13132F;
+$bg_color_icon_selector: #484357;
+$font_color_icon_selector: #DADADA;
+
+.post-icon { background-color: $bg_color_post_icon; }
+#current-icon-dropdown { background-color: $bg_color_post_icon; }
+#reply-icon-selector { background-color: $bg_color_icon_selector; color: $font_color_icon_selector; }
+
+$bg_color_post_character: #484357;
+$bg_color_post_screenname: #484357;
+$bg_color_post_author: #13132F;
+//$bg_color_post_character_narrow: $colors_lightbrown;
+//$bg_color_post_author_narrow: $colors_brown; // these are flipped from not-narrow mode to make the quick-switcher blend in // TODO: Should these flip on starries too?
+$font_color_post_info: #F3F3F3;
+
+.post-character { font-weight: bold; background-color: $bg_color_post_character; }
+.post-character, .post-character a, .post-character a:visited { color: $font_color_post_info; }
+.post-screenname { font-weight: bold; background-color: $bg_color_post_screenname; }
+.post-screenname, .post-screenname a, .post-screenname a:visited { color: $font_color_link_visited; }
+.post-author { font-weight: bold; background-color: $bg_color_post_author; }
+.post-author, .post-author a, .post-author a:visited { color: $font_color_post_info; }
+
+$font_color_paginate_link: #F3F3F3;
+$font_color_page_visited: $font_color_link_visited;
+$bg_color_page_current: #484357;
+$font_color_page_disabled: #58546B;
+
+.pagination .current { background-color: $bg_color_page_current; }
+.pagination a { color: $font_color_paginate_link; }
+.pagination a:visited { color: $font_color_page_visited; }
+.pagination .disabled { color: $font_color_page_disabled; }
+
+/* - icons */
+$border_color_icon_selected: #B9BBEB; // selected icon border
+$bg_color_icon: #D8D8D8;
+$bg_color_icon_credit: #D8D8D8;
+
+.replace-icon, .icons-box { background-color: $bg_color_icon; }
+.icon-credit { background-color: $bg_color_icon_credit; }
+
+#character-icon-selector, .time-loaded { color: #DADADA; }
+
+/* - tag items */
+$bg_color_tag_item: #2C2C5D;
+$font_color_tag_item: #444158;
+$font_color_tag_item_gallery: #F7F7F7;
+
+//TODO: This doesn't have even/odd tag items, but does have gallery-tags tag-item, should that be the case?
+tr.gallery-tags .tag-item { background-color: $bg_color_tag_item; color: $font_color_tag_item_gallery; }
+.tag-item { color: $font_color_tag_item; }
```

Comment:
yes – the contrast is only on the header which is a different color (`tr.gallery-tags`), I think, so this should be fine